crates/bevy_math/src/cubic_splines.rs

```diff
@@ -183,8 +185,8 @@ impl<P: Point> CubicGenerator<P> for CubicHermite<P> {
 /// Tangents are automatically computed based on the positions of control points.
 ///
 /// ### Continuity
-/// The curve is at minimum C0 continuous, meaning it has no holes or jumps. It is also C1, meaning the
-/// tangent vector has no sudden jumps.
+/// The curve is at minimum C1, meaning that it is continuous (it has no holes or jumps), and its tangent
```

Comment:
I'm interested in the use case where you want to have a spline with a smooth connected head and tail.

After this PR does that mean that it's enough to make `P_0` == `P_n`, since the spline then goes through both those? Will the tangents always align without discontinuities when the spline closes like that?

Reply:
There is no guarantee being made when connecting the endpoints like that as far as I can tell.

Reply:
That is correct; that could be something we support, but I think it'd be better to make a different CubicCurve builder for that, treating the control points as a cyclic list in the first place.

Reply:
Ok.  A separate builder (in some future PR) sounds great.
Thanks for clearing this up.

Reply:
I've been working on this a bit. Supporting closed splines is a priority of mine.

Reply:
@NthTensor  if you are working on this do you mind creating an issue in Bevy to track it and link it here?